app/assets/stylesheets/custom.scss

```diff
@@ -137,3 +137,20 @@ a {
 td {
   background-color: rgba(255, 255, 255, .5);
 }
+
+// variables, mixins
+@mixin box_sizing {
+  -moz-box-sizing:    border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing:         border-box;
+}
+
+/* miscellaneous */
+
+.debug_dump {
+  clear: both;
+  float: left;
+  width: 100%;
+  margin-top: 45px;
+  @include box_sizing;
```

Comment:
Name of mixin `box_sizing` should be written in all lowercase letters with hyphens instead of underscores